--- a/freebsd-include/sys/unistd.h
+++ b/freebsd-include/sys/unistd.h
@@ -1,80 +1,42 @@
-/*-
- * SPDX-License-Identifier: BSD-3-Clause
- *
- * Copyright (c) 1982, 1986, 1991, 1993
- *	The Regents of the University of California.  All rights reserved.
- *
- * Redistribution and use in source and binary forms, with or without
- * modification, are permitted provided that the following conditions
- * are met:
- * 1. Redistributions of source code must retain the above copyright
- *    notice, this list of conditions and the following disclaimer.
- * 2. Redistributions in binary form must reproduce the above copyright
- *    notice, this list of conditions and the following disclaimer in the
- *    documentation and/or other materials provided with the distribution.
- * 3. Neither the name of the University nor the names of its contributors
- *    may be used to endorse or promote products derived from this software
- *    without specific prior written permission.
- *
- * THIS SOFTWARE IS PROVIDED BY THE REGENTS AND CONTRIBUTORS ``AS IS'' AND
- * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
- * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
- * ARE DISCLAIMED.  IN NO EVENT SHALL THE REGENTS OR CONTRIBUTORS BE LIABLE
- * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
- * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS
- * OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
- * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
- * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
- * OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
- * SUCH DAMAGE.
- *
- *	@(#)unistd.h	8.3 (Berkeley) 1/21/94
- * $FreeBSD$
- */
-
 #ifndef _SYS_UNISTD_H_
 #define	_SYS_UNISTD_H_
 
 #include <sys/cdefs.h>
 #include <sys/_types.h>
 
-#ifndef _GID_T_DECLARED
-typedef	__gid_t		gid_t;
-#define	_GID_T_DECLARED
-#endif
-
-#ifndef _SIZE_T_DECLARED
-typedef	__size_t	size_t;
-#define	_SIZE_T_DECLARED
-#endif
-
-#ifndef _UID_T_DECLARED
-typedef	__uid_t		uid_t;
-#define	_UID_T_DECLARED
-#endif
-
 __BEGIN_DECLS
-int	 access(const char *, int);
-int	 chdir(const char *);
-int	 close(int);
-int	 dup(int);
-int	 dup2(int, int);
-int	 execl(const char *, const char *, ...);
-int	 execle(const char *, const char *, ...);
-int	 execlp(const char *, const char *, ...);
-int	 execv(const char *, char * const *);
-int	 execve(const char *, char * const *, char * const *);
-int	 execvp(const char *, char * const *);
-void	 _exit(int);
-int	 fork(void);
-long	 fpathconf(int, int);
-char	*getcwd(char *, size_t);
-gid_t	 getegid(void);
-uid_t	 geteuid(void);
-gid_t	 getgid(void);
-int	 getgroups(int, gid_t *);
-char	*getlogin(void);
-pid_t	 getpgrp(void);
-pid_t	 getpid(void);
-pid_t	 getppid(void);
-uid_t	 getuid(void);
-int	 isatty(int);
-int	 link(const char *, const char *);
-off_t	 lseek(int, off_t, int);
-long	 pathconf(const char *, int);
-int	 pause(void);
-int	 pipe(int *);
-ssize_t	 read(int, void *, size_t);
-int	 rmdir(const char *);
-int	 setgid(gid_t);
-int	 setpgid(pid_t, pid_t);
-pid_t	 setsid(void);
-int	 setuid(uid_t);
-unsigned int	 sleep(unsigned int);
-long	 sysconf(int);
-pid_t	 tcgetpgrp(int);
-int	 tcsetpgrp(int, pid_t);
-char	*ttyname(int);
-int	 unlink(const char *);
-ssize_t	 write(int, const void *, size_t);
+int access(const char *, int);
+int chdir(const char *);
+int close(int);
+int dup(int);
+int dup2(int, int);
+int execl(const char *, const char *, ...);
+int execle(const char *, const char *, ...);
+int execlp(const char *, const char *, ...);
+int execv(const char *, char * const *);
+int execve(const char *, char * const *, char * const *);
+int execvp(const char *, char * const *);
+void _exit(int);
+int fork(void);
+long fpathconf(int, int);
+char *getcwd(char *, size_t);
+gid_t getegid(void);
+uid_t geteuid(void);
+gid_t getgid(void);
+int getgroups(int, gid_t *);
+char *getlogin(void);
+pid_t getpgrp(void);
+pid_t getpid(void);
+pid_t getppid(void);
+uid_t getuid(void);
+int isatty(int);
+int link(const char *, const char *);
+off_t lseek(int, off_t, int);
+long pathconf(const char *, int);
+int pause(void);
+int pipe(int *);
+ssize_t read(int, void *, size_t);
+int rmdir(const char *);
+int setgid(gid_t);
+int setpgid(pid_t, pid_t);
+pid_t setsid(void);
+int setuid(uid_t);
+unsigned int sleep(unsigned int);
+long sysconf(int);
+pid_t tcgetpgrp(int);
+int tcsetpgrp(int, pid_t);
+char *ttyname(int);
+int unlink(const char *);
+ssize_t write(int, const void *, size_t);
 __END_DECLS
 
 #endif /* !_SYS_UNISTD_H_ */
